views/update/update-trusts-associated-with-overseas-entity.html

```diff
@@ -1,18 +1,14 @@
 {% extends "update-layout.html" %}
 
+{% set title = pageParams.title %}
 {% block pageTitle %}
-  Trusts associated with the overseas entity - {{ UPDATE_SERVICE_NAME }} - GOV.UK
+  {% include "includes/page-title.html" %}
```

Comment:
if you check the `includes/page-title.html` file, it sets the pageTitle with the Registration service name, so in our pages we don't want to use this includes for now so in your page title block you should have:

```
{% block pageTitle %}
  {% if errors and errors.errorList and errors.errorList.length > 0 %}
  Error:
  {% endif %}
  pageParams.title - {{ UPDATE_SERVICE_NAME }} - GOV.UK
{% endblock %}
```